alpha/nodejs-chart/templates/buildconfig.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.build.enabled }}
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: {{ include "nodejs.name" . }}
+  labels:
+    {{- include "nodejs.labels" . | nindent 4 }}
+spec:
+  source:
+    type: Git
+    git:
+      uri: {{ .Values.build.uri }}
+      ref: {{ .Values.build.ref }}
+{{- if .Values.build.contextDir }}
+    contextDir: {{ .Values.build.contextDir }}
+{{- end }}
+  strategy:
+    type: Source
+    sourceStrategy:
+      from:
+        kind: ImageStreamTag
+        namespace: openshift
+        name: nodejs:12
+  {{- if .Values.build.pullSecret }}
+      pullSecret:
+        name: {{ .Values.build.pullSecret }}
+  {{- end }}
+  {{- if .Values.build.env }}
+      env:
+        {{- tpl (toYaml .Values.build.env) . | nindent 8 }}
+  {{- end }}
+  output:
+    to:
+      kind: ImageStreamTag
+      name: {{ include "nodejs.imageName" . }}
```

Comment:
We should allow the image to be pushed to an ImageStreamTag or a DockerImage - https://github.com/redhat-developer/redhat-helm-charts/blob/master/alpha/quarkus-chart/templates/buildconfig.yaml#L9-L12

Reply:
I pushed a commit to make this change and used https://github.com/redhat-developer/redhat-helm-charts/commit/2202bec2bcd400c429205d767549067eb67acbc5 as a basis. PTAL.